docs/rst_man/src/contrib/sampler/daos/ldms-sampler_daos_sampler.rst

```diff
@@ -5,10 +5,9 @@ daos_sampler
 ===================
 
 :Date:   28 Apr 2022
```

Comment:
In all of the man pages, we need to move these metadata field lists need to be moved to _after_ the subtitle like they are in https://docutils.sourceforge.io/sandbox/manpage-writer/rst2man.txt. They aren't converting to man pages correctly as they currently are.

Reply:
Ah, I was following the ldmsd_decomposition.rst formatting so thank you for pointing this out! Man pages should be correct now.